projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1008,10 +1010,11 @@ private static function format_submission_data( $data ) {
 	/**
 	 * Helper function that display the error wrapper.
 	 *
+	 * @param array $classes - the classes to add to the error wrapper.
 	 * @return string HTML string for the error wrapper.
 	 */
-	private static function render_error_wrapper() {
-		$html  = '<div class="contact-form__error" data-wp-class--show-errors="state.showFormErrors">';
+	private static function render_error_wrapper( $classes = array() ) {
```

Comment:
Why an array instead of a string if we only ever pass it one class in the codebase? I would simplify, and if we later need to pass array we can turn it to a string higher up before this function.

Alternatively, pass `is_horizontal` boolean and manage CSS class directly in the function?

Reply:
> Why an array instead of a string

Just because is more future-proof. It can easily be a string. Using a boolean would not simplify as we'd need a ternary.